packages/test-e2e/README.md

```diff
@@ -0,0 +1,9 @@
+# Test E2E
```

Comment:
I've added this as a placeholder readme with minor script change for input/output paramaters